crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -118,9 +117,28 @@ impl ModuleLoader {
     }
   }
 
-  fn try_spawn_new_task(&mut self, info: &ResolvedRequestInfo) -> ModuleId {
+  fn try_spawn_new_task(
+    &mut self,
+    info: &ResolvedRequestInfo,
+    importer: Option<ImporterRecord>,
+  ) -> ModuleId {
     match self.visited.entry(Arc::<str>::clone(&info.path.path)) {
-      std::collections::hash_map::Entry::Occupied(visited) => *visited.get(),
+      std::collections::hash_map::Entry::Occupied(visited) => {
+        let id = *visited.get();
+        // If dependency already created, here need to update the `importers/dynamic_importers`
```

Comment:
What if the dependency is not created. Are we gonna missing some importers?

Reply:
No. Here has two case for `importer`.
1. Creating module with init `importer` at `normal_module_task`.
2. If the module already crated, here only need to update them.

Reply:
Assume the `init importer at normal_module_task` is still running and there are other depedencies that imports that running module. So, the module is not created currently. How we update them?

Reply:
I get it, here because the `dynamic_importers/importers` is invalid at `module_parsed` hook, it can be move to `module_loader`.